src/dnx/pal.h

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-void GetNativeBootstrapperDirectory(LPTSTR szPath);
+LPTSTR GetNativeBootstrapperDirectory();
```

Comment:
Actually, I would prefer to check this in as is if the cross platform string types haven't landed yet.